packages/trader/src/AppV2/Containers/Trade/__tests__/trade-types.spec.tsx

```diff
@@ -94,7 +94,7 @@ describe('TradeTypes', () => {
         render(mockTradeTypes());
 
         await userEvent.click(screen.getByText('View all'));
-        await userEvent.click(screen.getByText('Customize'));
+        await userEvent.click(screen.getByText('Customise'));
```

Comment:
Customise is no word , shouldn't be same "Customize"?

Reply:
It's from british variant of english, looks horrible, so it sounds, but seems like someone requested this change...